.github/workflows/cluster_endtoend_onlineddl_revert_mysql80.yml

```diff
@@ -38,7 +38,8 @@ jobs:
             - 'test.go'
             - 'Makefile'
             - 'build.env'
-            - 'go.[sumod]'
+            - 'go.sum'
+            - 'go.mod'
```

Comment:
Running the ci workflow generator changed a bunch of other tests too, but that is unrelated to this PR.

Reply:
sounds good